src/System.Collections.NonGeneric/tests/ArrayList/PropertyIsSynchronizedTests.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Collections;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Collections.ArrayListTests
+{
+    public class GetIsSynchronizedTests
+    {
+        private ArrayList _alst2;
+        private Int32 _iNumberOfElements = 10;
+
+        [Fact]
+        public void TestGetIsSynchronized()
+        {
+            ArrayList alst1;
+            Int32 iNumberOfTimes;
+            String strValue;
+
+            Task[] workers;
+            Action ts1;
+            Int32 iNumberOfWorkers = 10;
+
+            //[] Vanila test case - Synchronized returns an IList that is thread safe
+            // We will try to test this by getting a number of threads to write some items
+            // to a synchronized IList
+
+            alst1 = new ArrayList();
+            _alst2 = ArrayList.Synchronized(alst1);
+
+            workers = new Task[iNumberOfWorkers];
+            ts1 = new Action(AddElements);
+            // LongRunning will have problem when run in xUnit
+            for (int i = 0; i < workers.Length; i++)
+            {
+                workers[i] = Task.Factory.StartNew(ts1, TaskCreationOptions.LongRunning);
```

Comment:
This should be:
Task.Factory.StartNew(ts1, CancellationToken.None, TaskCreationOptions.LongRunning, TaskScheduler.Default);
assuming you actually want it to have its own dedicated thread.  If not, you'd just make it:
Task.Run(ts1);